userguide.md

```diff
@@ -0,0 +1,50 @@
+# USER GUIDE
+
+## Smaily E-Commerce extension for Magento 2.0+
+>[Smaily](https://smaily.com/) is an intentionally simple tool for sending beautiful email newsletters. 
+
+## Purchase / Download
+
+1. You can purchase **Smaily E-Commerce extension for Magento 2.0+** from [Magento marketplace](https://marketplace.magento.com/smaily-smailyformagento.html) or download it from our [github repository](https://github.com/sendsmaily/smaily-magento-extension).
```

Comment:
`github repository` should be written as `GitHub repository`.

Reply:
Changed to GitHub https://github.com/sendsmaily/smaily-magento-extension/pull/34/commits/ce76c4eb0d9c5088068b7af75c6bdcf4b874b3d2